AdminWebsite/AdminWebsite.AcceptanceTests/appsettings.json

```diff
@@ -10,7 +10,7 @@
         "GraphApiBaseUri": "https://graph.microsoft.com/",
         "RedirectUri": " https://localhost:5400/login",
         "PostLogoutRedirectUri": "https://localhost:5400/",
-        "ResourceId": "resourceID"
+        "ResourceId": "98059e35-6413-4058-8432-3a0c76c32da0z"
```

Comment:
ResourceID should be secret, dont need this change

Reply:
sorry I will revert it